scripts/webpack/webpack.config.ts

```diff
@@ -68,17 +62,21 @@ const webpackConfig: webpack.Configuration = {
   },
   plugins: [
     new ForkTsCheckerWebpackPlugin({
-      tsconfig: paths.docs('tsconfig.json'),
-      watch: [paths.docsSrc(), paths.packages()],
+      typescript: {
+        configFile: paths.docs('tsconfig.json'),
+      },
+      // watch: [paths.docsSrc(), paths.packages()],
```

Comment:
Does this break anything in the v0 dev experience?

Reply:
This option is completely removed. I'm guessing it has been changed to be automated (via the watch api internally?) That option was removed 2 major's back...

Reply:
removed.